src/pause-play/js/main.js

```diff
@@ -66,18 +79,21 @@ class PeerConnection {
       this.remoteView.srcObject = e.stream;
     };
 
-    var onCreateOfferSuccess = this.onCreateOfferSuccess.bind(this);
     this.localConnection
       .createOffer({offerToReceiveAudio: 1, offerToReceiveVideo: 1})
-      .then(onCreateOfferSuccess, logError);
+      .then((desc) => {
+        this.onCreateOfferSuccess(desc);
+      }, logError);
   }
 
   onCreateOfferSuccess(desc) {
     this.localConnection.setLocalDescription(desc);
     this.remoteConnection.setRemoteDescription(desc);
 
-    var onCreateAnswerSuccess = this.onCreateAnswerSuccess.bind(this);
-    this.remoteConnection.createAnswer().then(onCreateAnswerSuccess, logError);
+    this.remoteConnection.createAnswer().then(
+      () => {
```

Comment:
If I do that, it complains:

  94:10  error  'desc' is already declared in the upper scope                                                                  no-shadow
  94:19  error  Statement inside of curly braces should be on next line                                                        brace-style
  94:52  error  Closing curly brace should be on the same line as opening curly brace or on the line after the previous block  brace-style